File: java/src/main/java/com/fengtian/qianer/atcoder/abc/abc131/ABC131_A.java
package com.fengtian.qianer.atcoder.abc.abc131;

import java.util.Scanner;

public class ABC131_A {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        final char[] S = sc.next().toCharArray();

        if (S[0] == S[1] || S[1] == S[2] || S[2] == S[3]) {
            System.out.println("Bad");
        } else {
            System.out.println("Good");
        }
    }
}

File: kotlin/src/abc/abc131/ABC131_A.kt
package abc.abc131

import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    val s = sc.next()

    for (x in 1 until s.length) {
        if (s[x] === s[x-1]) {
            println("Bad")
            return
        }
    }
    println("Good")
}

